homeassistant/components/rympro/const.py

```diff
@@ -1,3 +1,13 @@
 """Constants for the Read Your Meter Pro integration."""
+from enum import Enum
 
 DOMAIN = "rympro"
+
+
+class MeterSensor(Enum):
```

Comment:
I think this should be StrEnum
```suggestion
class MeterSensor(StrEnum):
```

Reply:
`Enum` is part of the standard library, while `StrEnum` isn't. What's the benefit of using it?

Reply:
All the values are strings, so I think it makes sense regardless.